python/test/test_polars.py

```diff
@@ -532,18 +533,21 @@ def test_polars_threshold():
         .collect()
     )
 
-
+@pytest.mark.skip(reason="setting OPENDP_POLARS_LIB_PATH interferes with the execution of other tests")
```

Comment:
It feels like one consequence of this PR is that a test that used to run will no longer be run? That doesn't seem like the right direction, but I might be missing something.

Reply:
There was one new test added in this PR, and it gets skipped by default. I don't have a good way to test this functionality within our current CI because it causes race issues/failures in other tests, when tests are run in parallel.

Reply:
It's a bit kludgy, but in smoke-test.yml maybe run everything except this test, and then run just this test?
```
pytest --nbmake source/**/*.ipynb -n=auto -k 'not test_replace_binary_path'
pytest -k 'test_replace_binary_path'
```

Reply:
I think pytest should run out-of-the box without special commands. Pushing a different approach.